src/app/WriteHandler.h

```diff
@@ -189,6 +189,9 @@ class WriteHandler : public Messaging::ExchangeDelegate
     CHIP_ERROR WriteClusterData(const Access::SubjectDescriptor & aSubject, const ConcreteDataAttributePath & aPath,
                                 TLV::TLVReader & aData);
 
+    // Checks if the given path corresponds to a list attribute
```

Comment:
This needs some documentations about the caveats, like what this method does if it does not know the answer.  It would be worth also rephrasing the comment to something like "is known for sure to correspond to a list attribute".

Reply:
Seems reasonable (especially regarding the comment before). Will make it return optional/ternary (basically know for sure yes/now and unknown)